pyEpiabm/pyEpiabm/__init__.py

```diff
@@ -0,0 +1,4 @@
+from .person import Person  # noqa
```

Comment:
Add some heading at the top to explain the purpose of the file. Try following the ``branchpro`` former with the licensing ad stuff?